Plugins/Public/pvecontroller/Main.cpp

```diff
@@ -572,6 +600,26 @@ void __stdcall HkCb_AddDmgEntry(DamageList *dmg, unsigned short p1, float damage
 						iBountyPayout *= itSystemScale->second;
 				}
 
+				// Multiply by class diff multiplier if applicable.
+				if (iLoadedClassDiffMultipliers) {
+					uint iDmgFromShipClass = Archetype::GetShip(Players[iDmgFrom].iShipArchetype)->iShipClass;
+
+					int classDiff = 0;
+					map<uint, uint>::iterator itVictimType = mapShipClassTypes.find(victimShiparch->iShipClass);
+					map<uint, uint>::iterator itKillerType = mapShipClassTypes.find(iDmgFromShipClass);
+					if (itVictimType != mapShipClassTypes.end() && itKillerType != mapShipClassTypes.end())
+						classDiff = itVictimType->second - itKillerType->second;
+
+					map<int, float>::iterator itDiffMultiplier = mapClassDiffMultipliers.lower_bound(classDiff);
+					if (itDiffMultiplier != mapClassDiffMultipliers.end())
+						if (itDiffMultiplier->second != 1.0f) //We only need to modify the payout if the multiplier is not 1.
```

Comment:
to avoid a multiplication by 1...? think we can skip this check

Reply:
Removed in latest commit.